Pocket.Logger/Core/LogEvents.cs

```diff
@@ -83,11 +83,11 @@ internal static partial class LogEvents
         private static void SubscribeLoggers<T>(
             IEnumerable<Type> pocketLoggerTypes,
             LoggerSubscription subscription,
-            Action<T> onEntryPosted = null)
+            Action<T> onEntryPosted)
         {
             foreach (var loggerType in pocketLoggerTypes.Distinct())
             {
-                if (onEntryPosted != null)
+                if (onEntryPosted is not null)
```

Comment:
Can this null check be removed now that `onEntryPosted` no longer has a null default value? It seems pointless to support calling this method with `onEntryPosted = null`....